### PR TITLE
ft/ZENKO-422 setting up end to end tests

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: charts/cloudserver-front/values.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+ci:
+  enabled: true
```
Comment (ssalaues) [inline charts/cloudserver-front/values.yaml]: Lets disable this by default

**Comment by sylvain-killian** [comment]
having a `bootstrap` stage reported in test results is unexpected (only pre-merge expected), please fill in a bug report on EVE.

File: eve/main.yml [hunk 2]
```diff
@@ -24,6 +50,11 @@ stages:
         namespaces:
         - "testNamespace"    # <<< the default namespace for that stage
     steps:
+    - ShellCommand:
+        name: Install curl
+        command: |-
+          apk --update upgrade
+          apk --update add curl
```
Comment (sylvain-killian) [inline eve/main.yml]: this must be cached in the worker definition to avoid the extra internet dependency (flakiness control)

Comment (tcarmet) [reply eve/main.yml]: There's a pull request regarding https://github.com/scality/zenko-releng/pull/2

Waiting for merge and approvals

**Pull request merged**